cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/NewCommand.java

```diff
@@ -71,9 +71,9 @@ public void execute() {
         // Check if the project name is given
         if (null == argList) {
             CommandUtil.printError(errStream,
-                    "The following required arguments were not provided:\n" +
+                    "following required arguments were not provided:\n" +
                          "    <project-name>",
-                    "ballerina new <project-name>",
+                    "ballerina new <project-name>.",
```

Comment:
This is the usage IINM. I don't think we should add a full stop at the end, we could expect someone to copy and use this.

Reply:
Removed the fullstop.